includes/html/pages/device/showconfig.inc.php

```diff
@@ -318,16 +318,18 @@
     if (!empty($text)) {
         if (isset($previous_config)) {
             $language = 'diff';
+        } elseif ($device['os'] == 'pfsense') {
```

Comment:
Should we move this to the device definition? So have `config_language: xml` in the pfsense definition then fall back to ios for all others?

Reply:
config_highlighting would be my naming

Reply:
Even better :)

Reply:
@brownowski Can you implement this please?